docs/source/algorithms/LoadISISNexus-v2.rst

```diff
@@ -14,13 +14,118 @@ Loads a Nexus file created from an ISIS instrument.
 Data loaded from Nexus File
 ###########################
 
-The nexus file must have ``/raw_data_1`` as its main group and
+The nexus file must have ``raw_data_1`` as its main group and
 contain a ``/isis_vms_compat`` group to be loaded.
 
-The workspace data is loaded from ``/raw_data_1/Detector_1``. 
-Instrument information is loaded ``/raw_data_1/Instrument``, if available there and not overriden.
+The workspace data is loaded from ``raw_data_1/Detector_1``. 
+Instrument information is loaded ``raw_data_1/Instrument``, if available there and not overriden.
 Also the ``NSP1``, ``UDET``, ``SPEC``, ``HDR``, ``IRPB``, ``RRPB``, ``SPB`` and ``RSPB`` sections of
-``/raw_data_1/isis_vms_compat`` are read.
+``raw_data_1/isis_vms_compat`` are read.
+
+Here are some tables that show it in more detail:
+
++------------------------------+-------------------------------------------+-------------------------------------+
+| Description of Data          | Found in Nexus file                       | Placed in Workspace (Workspace2D)   |
+|                              | (within 'raw_data_1')                     |                                     |
++==============================+===========================================+=====================================+
+| Monitor Data                 | within groups of Class NXMonitor          | Depending on property LoadMonitors, | 
+|                              | (one for each monitor)                    | monitor histogram data              |
```

Comment:
Maybe when it says "one for each monitor" it could clarify that the groups are named `monitor_1`, `monitor_2`, etc.